screens/HotspotFormScreen.tsx

```diff
@@ -571,6 +656,7 @@ const AddLocation = ({
       <InputField
         placeholder="Adresă"
         multiline={true}
+        maximumLength={255}
```

Comment:
you should use the constant from `inputLimits.ts`

Reply:
Done 💯